<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Container Breaker</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="indentation">
    <number>20</number>
   </property>
   <property name="animated">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>160</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Offset</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Size</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Compressed Size</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arguments</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>0</y>
     <width>141</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="bOpen">
      <property name="text">
       <string>Open Container</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bClear">
      <property name="text">
       <string>Clear Tree</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bBuild">
      <property name="text">
       <string>Build Container</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>90</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="bFAdd">
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bFExt">
      <property name="text">
       <string>Extract File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bFReplace">
      <property name="text">
       <string>Replace File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bFDel">
      <property name="text">
       <string>Delete File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
